swap_meet/vendor.py

```diff
@@ -1,2 +1,80 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self,inventory=None):
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
```

Comment:
We could use a ternary here, like for `id` in the `Item` class, to make this more concise:
```suggestion
            self.inventory = inventory if inventory else None
```